configure.ac

```diff
@@ -406,6 +406,26 @@ case "${host_cpu}" in
 esac
 AC_DEFINE_UNQUOTED([CPU_SPINWAIT], [$CPU_SPINWAIT])
 
+AC_MSG_CHECKING([number of significant virtual address bits])
+case "${host_cpu}" in
+  x86_64|aarch64)
+    LG_VADDR=48
```

Comment:
Just FYI this is changing soon to 57:

https://software.intel.com/sites/default/files/managed/2b/80/5-level_paging_white_paper.pdf